src/EtwHeapDump/EtwHeapDump.csproj

```diff
@@ -28,8 +28,18 @@
     </FilesToSign>
 
     <PackageReference Include="MicroBuild.Core" Version="0.2.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
```

Comment:
Is this only needed when using the local copy of ClrMD?  I would expect this to be pulled in via NuGet.  If so, maybe we put this under the local condition?

Reply:
Yes, this is needed only for local ClrMD.

if I don't add this and built with local flipped on, the build fails with some version mismatch warnings.

It feels like Nuget pulled the wrong version and then complain the pull was wrong, so I added it to make sure Nuget pull the right one.

Reply:
Gotcha.  In that case, what do you think about putting this under the local condition, so that the this doesn't apply when we use NuGet?

Reply:
Sure